tests/components/sensorpush_cloud/test_config_flow.py

```diff
@@ -0,0 +1,92 @@
+"""Test the SensorPush Cloud config flow."""
+
+from __future__ import annotations
+
+from unittest.mock import AsyncMock
+
+import pytest
+from sensorpush_ha import SensorPushCloudError
+
+from homeassistant import config_entries
+from homeassistant.components.sensorpush_cloud.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from .const import MOCK_DATA
+
+
+@pytest.mark.data(MOCK_DATA)
+async def test_form(
+    hass: HomeAssistant,
+    mock_api: AsyncMock,
+    mock_helper: AsyncMock,
+    mock_setup_entry: AsyncMock,
+) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == FlowResultType.FORM
```

Comment:
compare `enums` with `is` (the rest below as well)
```suggestion
    assert result["type"] is FlowResultType.FORM
```